From dea72b00026c6d815ceb610574e43ffc7bc1e71f Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanap23@in.ibm.com>
Date: Wed, 15 Feb 2023 12:51:11 +0000
Subject: [PATCH] Updated protobuf to 3.19.6

---
 WORKSPACE | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index 238f2b6c9..02d046162 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -111,11 +111,11 @@ sass_repositories()
 # might lead to test failures.
 http_archive(
     name = "com_google_protobuf",
-    sha256 = "1fbf1c2962af287607232b2eddeaec9b4f4a7a6f5934e1a9276e9af76952f7e0",
-    strip_prefix = "protobuf-3.9.2",
+    sha256 = "387e2c559bb2c7c1bc3798c4e6cff015381a79b2758696afcbf8e88730b47389",
+    strip_prefix = "protobuf-3.19.6",
     urls = [
-        "http://mirror.tensorflow.org/github.com/protocolbuffers/protobuf/archive/v3.9.2.tar.gz",
-        "https://github.com/protocolbuffers/protobuf/archive/v3.9.2.tar.gz",  # 2019-09-23
+        "http://mirror.tensorflow.org/github.com/protocolbuffers/protobuf/archive/v3.19.6.zip",
+        "https://github.com/protocolbuffers/protobuf/archive/v3.19.6.zip",  # 2019-09-23
     ],
 )
 
-- 
2.34.1

